src/bar_chart.py

```diff
@@ -0,0 +1,92 @@
+import pandas as pd
+import altair as alt
+
+def bar_chart_gen(data, genrelist = []):
+    """
+    Generate the horizontal bar chart to show top actors in the top rated movies.
+
+    Parameters
+    ----------
+    data : pandas dataframe
+        Dataframe containing the data to plot.
+    genrelist : list
+        List of genres to filter for.
+    
+    Results
+    -------
+    chart : html of altair Chart
+        The generated bar chart converted to html
+    """
+    alt.renderers.set_embed_options(theme='dark')
+    x = 'averageRating'
+    y = 'primaryName'
+
+    # filtering data for genres
+    if genrelist != []:
```

Comment:
I don't think this is necessary because data is already filtered in app.py?

Reply:
Yes, agreed. I had added the code for this and then saw your updated app.py where you added the filtered-data portion (brilliant idea). It is definitely redundant so I will go ahead and remove.